pkg/cluster/base.go

```diff
@@ -133,6 +133,13 @@ type CreateClusterProperties struct {
 	CreateClusterBanzaiCloud *banzaicloud.CreateClusterBanzaiCloud `json:"clusterTopology,omitempty" yaml:"clusterTopology,omitempty"`
 }
 
+// ClusterBootstrapInfo for PKE provisioning
+type ClusterBootstrapInfo struct {
```

Comment:
This struct is not used anywhere, but in the API. Please move it there ("model-package" is an anti-pattern).